test/karma.conf.js

```diff
@@ -1,17 +1,58 @@
+var babel = require('rollup-plugin-babel');
+var commonjs = require('rollup-plugin-commonjs');
+var json = require('rollup-plugin-json');
+var multiEntry = require('rollup-plugin-multi-entry');
+var resolve = require('rollup-plugin-node-resolve');
+var string = require('rollup-plugin-string');
+
 module.exports = function(config) {
   config.set({
     basePath: '..',
     frameworks: ['qunit'],
-    files: [
-      'node_modules/sinon/pkg/sinon.js',
-      'test/dist/bundle.js'
-    ],
     browsers: ['ChromeHeadless'],
+    client: {
+      clearContext: false,
+      qunit: {
+        showUI: true,
+        testTimeout: 5000
+      }
+    },
+    files: [{
+      included: false,
+      pattern: 'src/**/*.js',
+      watched: true
+    }, {
+      pattern: 'test/**/*.test.js',
+      watched: false
```

Comment:
I guess rollup does this for us?

Reply:
the karma rollup preprocessor does. Would be good to add a comment here explaining why the above are what it is.